.gitignore

```diff
@@ -1,11 +1,21 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python-without-pypy
+*.egg-info
+*.profraw
+*.pyc
+*.pyo
+.coverage
 .installed.cfg
+.mr.developer.cfg
+.tox/
 bin/
-develop-eggs/
-src/zc.beforestorage.egg-info/
-parts/
-*.py[co]
-__pycache__/
```

Comment:
Huh, the template doesn't ignore `__pycache__/`?  Well, I guess everything inside will be caught by the `*.pyc`, `*.pyo` rules.

Reply:
Fixed in zopefoundation/meta.